snapshot/snapshotfs/source_snapshots.go

```diff
@@ -40,7 +40,7 @@ func (s *sourceSnapshots) Sys() interface{} {
 }
 
 func (s *sourceSnapshots) ModTime() time.Time {
-	return time.Now()
+	return s.rep.Time()
```

Comment:
BTW this is completely arbitrary, perhaps (separate PR) we could use last repo sync time?